api/clear_admin_knowledge.py

```
@@ -0,0 +1,167 @@
+
+    matching_ids: list[str] = []
+    offset = 0
+    while offset < total:
+        batch = collection.get(
+            limit=BATCH_SIZE,
+            offset=offset,
+            include=["metadatas"],
+        )
+        ids = batch.get("ids") or []
+        metadatas = batch.get("metadatas") or []
+        for doc_id, metadata in zip(ids, metadatas):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)